docs/user-namespace.md

```diff
@@ -8,11 +8,11 @@ URL: http://madrid.gobierto.dev/user/login
 
 Available users:
 
-| Email               | Password | Notes                                                         |
-| ---                 | ---      | ---                                                           |
-| dennis@gobierto.dev | gobierto | regular user, madrid.gobierto.dev as source site              |
-| reed@gobierto.dev   | gobierto | regular user, unconfirmed, madrid.gobierto.dev as source site |
-| susan@gobierto.dev  | gobierto | regular user, santander.gobierto.dev as source site           |
+| Email               | Password | Notes                                                                        |
+| ---                 | ---      | ---                                                                          |
+| dennis@gobierto.dev | gobierto | regular user, madrid.gobierto.dev as source site                             |
```

Comment:
Just a stupid thing:

- dennis row should mention he's verified against census
- susan row should mention she's not verified

I can take care of the change.